lib/upload.js

```diff
@@ -81,6 +83,27 @@ class Upload {
       return;
     }
 
+    // In React Native, when user selects a file, instead of a File or Blob, 
+    // you usually get a file object {} with a uri property that contains
+    // a local path to the file. We use XMLHttpRequest to fetch 
+    // the file blob, before uploading with tus.
+    if (isReactNative && 
+      typeof file === 'object' && 
+      file !== null) {
+        console.log('IT’S FUCKING WORKING!!!!!!')
```

Comment:
lets keep this :wink:

Reply:
Been rapidly learning how to undo this for the past 30 minutes.